README.md

```diff
@@ -7,29 +7,26 @@ A minor Emacs mode for running
 [Clojure test
 mode](https://github.com/technomancy/clojure-mode/blob/master/clojure-test-mode.el).
 
+Note that this is designed for classic expectations, and will not work with the
```

Comment:
I don't think this is correct: it was designed for regular `clojure.test` and it used to work just fine with this version of Expectations. There is a separate Emacs mode for classic Expectations: https://github.com/clojure-expectations/expectations-mode

However, that link (to `clojure-test-mode`) is invalid now and I think Emacs users run tests via https://github.com/clojure-emacs/clojure-mode/blob/master/clojure-mode.el instead? (I haven't used Emacs for years tho').

Reply:
I misread which readme you were proposing to change. My bad. Ignore my comments!